pyseir/inference/infer_rt.py

```diff
@@ -513,77 +512,33 @@ def infer_all(self, plot=True, shift_deaths=0):
         if plot and df_all is not None:
             plt.figure(figsize=(10, 6))
 
-            if "Rt_ci5__new_deaths" in df_all:
-                plt.fill_between(
-                    df_all.index,
-                    df_all["Rt_ci5__new_deaths"],
-                    df_all["Rt_ci95__new_deaths"],
-                    alpha=0.2,
-                    color="firebrick",
-                )
-                plt.scatter(
-                    df_all.index,
-                    df_all["Rt_MAP__new_deaths"].shift(periods=shift_deaths),
-                    alpha=1,
-                    s=25,
-                    color="firebrick",
-                    label="New Deaths",
-                )
-
-            if "Rt_ci5__new_cases" in df_all:
-                plt.fill_between(
-                    df_all.index,
-                    df_all["Rt_ci5__new_cases"],
-                    df_all["Rt_ci95__new_cases"],
-                    alpha=0.2,
-                    color="steelblue",
-                )
-                plt.scatter(
-                    df_all.index,
-                    df_all["Rt_MAP__new_cases"],
-                    alpha=1,
-                    s=25,
-                    color="steelblue",
-                    label="New Cases",
-                    marker="s",
-                )
-
-            if "Rt_ci5__new_hospitalizations" in df_all:
-                plt.fill_between(
-                    df_all.index,
-                    df_all["Rt_ci5__new_hospitalizations"],
-                    df_all["Rt_ci95__new_hospitalizations"],
-                    alpha=0.4,
-                    color="darkseagreen",
-                )
-                plt.scatter(
-                    df_all.index,
-                    df_all["Rt_MAP__new_hospitalizations"],
-                    alpha=1,
-                    s=25,
-                    color="darkseagreen",
-                    label="New Hospitalizations",
-                    marker="d",
-                )
-            if "Rt_MAP_composite" in df_all:
```

Comment:
It would be great if we could keep this line, as we have been using these plots to debug